app/views/conversation_messages/_message.html.erb

```diff
@@ -1,9 +1,9 @@
-<div class="flex flex-col <%= message.user == user ? "col-start-2 col-end-5" : "col-start-1 col-end-4" %>" id="<%= dom_id(message)%>">
-  <div class="flex-1 p-4 rounded-t-full <%= message.user == user ? "bg-green-600 text-white rounded-bl-full" : "bg-gray-200 text-gray-900 rounded-t-full rounded-br-full" %>">
+<div class="flex flex-col <%= message.user == user ? "col-start-2 col-end-5" : "col-start-1 col-end-4" %>" id="<%= dom_id(message)%>" data-hidden="<%= owned_by %>">
```

Comment:
para saan itong `data-hidden`?

Reply:
ayan yung way para ma display:none ko sa screen ng sender yung bubble message, ang ginawa ko kasing workaround is kada send ng message, nagreredirect. Then, with that data-hidden, hidden sa sender yung sinend na message via websocket. 

So magrerender lang yung partial template data na sinend via websocket sa screen ng receiver. Dahil naka display:none yung sa sender side, need ko iredirect kada send ng message para makita niya yung sinend niyang message.

Reply:
oooooh nice good job pre! galing galing